moveit_simple/test/moveit_simple.cpp

```diff
@@ -62,7 +62,21 @@ TEST(MoveitSimpleTest, add_trajectory)
   EXPECT_TRUE(robot.addTrajPoint(TRAJECTORY_NAME, "waypoint2", 3.0));
   EXPECT_TRUE(robot.addTrajPoint(TRAJECTORY_NAME, "waypoint3", 4.0));
 
-  EXPECT_TRUE(robot.execute(TRAJECTORY_NAME));
+  // Check if a proper plan and execution works.
+  moveit_simple::TrajectoryGoal trajectory_goal;
+  EXPECT_TRUE(robot.plan(TRAJECTORY_NAME, trajectory_goal));
```

Comment:
Is it like
1. planAndExecute(traj_name)
2. plan(traj_name, trajectory_goal) and execute(trajectory_goal)?